src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -47,7 +47,7 @@
 public class ExprRawString extends SimpleExpression<String> {
 
 	static {
-		Skript.registerExpression(ExprRawString.class, String.class, ExpressionType.SIMPLE, "raw %strings%");
+		Skript.registerExpression(ExprRawString.class, String.class, ExpressionType.COMBINED, "raw %strings%");
```

Comment:
Combined is when the syntax contains two or more expressions such as raw %strings% in %strings%

Reply:
so should it be property?

Reply:
A property is when `something of %object%` or `%object%'[s] something` so it's fine as a simple.

Reply:
@UnderscoreTud Lime's wrong on this, it should be COMBINED

Reply:
😭 😭